vllm/attention/ops/prefix_prefill.py

```diff
@@ -2,12 +2,13 @@
 # https://github.com/ModelTC/lightllm/blob/main/lightllm/models/llama/triton_kernel/context_flashattention_nopad.py
 
 import torch
-import triton
-import triton.language as tl
 
 from vllm.platforms import current_platform
+from vllm.triton_utils import maybe_import_triton
 
-if triton.__version__ >= "2.1.0":
+triton, tl = maybe_import_triton()
+
+if triton.__version__ >= "2.1.0" or triton.__version__ == "0.0.0":
```

Comment:
I'm not sure we really need this check at all because there is no `else` clause. If `triton.__version__ < "2.1.0"` then there are runtime errors due to the below code being missing.